libs/base/Data/Vect.idr

```diff
@@ -87,6 +87,14 @@ take : (n  : Nat)
 take 0 xs = Nil
 take (S k) (x :: xs) = x :: take k xs
 
+namespace Stream
+  ||| Take precisely n elements from the stream.
+  ||| @ n how many elements to take
+  ||| @ xs the stream
+  public export
+  take : (n : Nat) -> (xs : Stream a) -> Vect n a
+  take = fromList . take
```

Comment:
i'm not clear on the implications of using `fromList` instead of copy-pasting from the `List` implementation. Anyone know?